shopping-cart example fails to compile

`mvn clean compile test-compile` fails with the following errors on the current `master`:
```
[INFO] 30 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for kotlin-support 0.4.3:
[INFO] 
[INFO] cloudstate-kotlin-support .......................... SUCCESS [ 14.965 s]
[INFO] shopping-cart ...................................... FAILURE [  1.207 s]
[INFO] kotlin-pingpong .................................... SKIPPED
[INFO] kotlin-chat ........................................ SKIPPED
[INFO] kotlin-support ..................................... SKIPPED
[INFO] examples ........................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  16.504 s
[INFO] Finished at: 2020-04-20T12:56:01-07:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (java-compile) on project shopping-cart: Compilation failure: Compilation failure: 
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/io/cloudstate/Eventing.java:[26,7] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class io.cloudstate.Eventing
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/google/protobuf/Empty.java:[34,7] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.google.protobuf.Empty
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/persistence/Domain.java:[66,9] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.example.shoppingcart.persistence.Domain.LineItem
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/persistence/Domain.java:[810,9] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.example.shoppingcart.persistence.Domain.ItemAdded
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/persistence/Domain.java:[1428,9] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.example.shoppingcart.persistence.Domain.ItemRemoved
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/persistence/Domain.java:[2005,9] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.example.shoppingcart.persistence.Domain.Cart
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/google/api/HttpBody.java:[59,7] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.google.api.HttpBody
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[77,9] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.example.shoppingcart.Shoppingcart.AddLineItem
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[952,9] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.example.shoppingcart.Shoppingcart.RemoveLineItem
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[1633,9] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.example.shoppingcart.Shoppingcart.GetShoppingCart
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[2204,9] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.example.shoppingcart.Shoppingcart.LineItem
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[2956,9] cannot find symbol
[ERROR]   symbol:   class UnusedPrivateParameter
[ERROR]   location: class com.example.shoppingcart.Shoppingcart.Cart
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/io/cloudstate/EventsProto.java:[51,7] no suitable method found for internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[])
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],java.lang.Class<?>,java.lang.String[],java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/io/cloudstate/Eventing.java:[23,3] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/io/cloudstate/EntityKey.java:[44,7] no suitable method found for internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[])
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],java.lang.Class<?>,java.lang.String[],java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/google/protobuf/Empty.java:[31,3] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/google/protobuf/EmptyProto.java:[38,7] no suitable method found for internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[])
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],java.lang.Class<?>,java.lang.String[],java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/google/api/HttpBodyProto.java:[40,7] no suitable method found for internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[])
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],java.lang.Class<?>,java.lang.String[],java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/persistence/Domain.java:[63,5] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/persistence/Domain.java:[807,5] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/persistence/Domain.java:[1425,5] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/persistence/Domain.java:[2002,5] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/persistence/Domain.java:[2789,7] no suitable method found for internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[])
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],java.lang.Class<?>,java.lang.String[],java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/google/api/HttpBody.java:[56,3] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[74,5] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[949,5] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[1630,5] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[2201,5] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[2953,5] method does not override or implement a method from a supertype
[ERROR] /home/vkorenev/Projects/Cloudstate/kotlin-support/examples/shopping-cart/target/generated-sources/protobuf/java/com/example/shoppingcart/Shoppingcart.java:[3757,7] no suitable method found for internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[])
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR]     method com.google.protobuf.Descriptors.FileDescriptor.internalBuildGeneratedFileFrom(java.lang.String[],java.lang.Class<?>,java.lang.String[],java.lang.String[],com.google.protobuf.Descriptors.FileDescriptor.InternalDescriptorAssigner) is not applicable
[ERROR]       (actual and formal argument lists differ in length)
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :shopping-cart
```

--- a/cloudstate-kotlin-support/pom.xml
+++ b/cloudstate-kotlin-support/pom.xml
@@ -37,18 +37,6 @@
             <groupId>io.cloudstate</groupId>
             <artifactId>cloudstate-java-support</artifactId>
             <version>0.4.3</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>com.google.protobuf</groupId>
-                    <artifactId>protobuf-java</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-
-        <dependency>
-            <groupId>com.google.protobuf</groupId>
-            <artifactId>protobuf-java</artifactId>
-            <version>3.5.1</version>
         </dependency>
 
         <dependency>

